Domains/4-Application/Connectors/CivilInfrastructureFramework/CifBridge.ecschema.xml

```diff
@@ -535,7 +542,7 @@
     <ECEntityClass typeName="BeamSegmentCollectionBaseAspect" modifier="Abstract">
         <BaseClass>cifcmn:CivilPresentation</BaseClass>
         <ECProperty propertyName="BeamSegmentCollection_BeamDefinition" typeName="string" displayLabel="Beam Definition" category="BridgeModelerAttributes_Beam" priority="299999"/>
-        <ECProperty propertyName="BeamSegmentCollection_Direction" typeName="double" displayLabel="Direction" category="BridgeModelerAttributes_Beam" priority="299998" kindOfQuantity="cifu:ANGLE"/>
+        <ECProperty propertyName="BeamSegmentCollection_Direction" typeName="double" displayLabel="Direction" category="BridgeModelerAttributes_Beam" priority="299998" kindOfQuantity="cifu:BEARING"/>
```

Comment:
The KOQ changed here, but the underlying persistence unit of both the old and new KOQ is the same. So, I'm assuming I don't need to add a unit change custom attribute in this case.